tools/postinstall-android.js

```diff
@@ -98,7 +98,7 @@ if (data.indexOf("SalesforceHybrid") < 0)
     var newAndroidDepTree = "android {\n\tpackagingOptions {\n\t\texclude 'META-INF/LICENSE'\n\t\texclude 'META-INF/LICENSE.txt'\n\t\texclude 'META-INF/DEPENDENCIES'\n\t\texclude 'META-INF/NOTICE'\n\t}";
     replaceTextInFile(path.join(appProjectRoot, 'build.gradle'), oldAndroidDepTree, newAndroidDepTree);
     var oldGradleToolsVersion = "com.android.tools.build:gradle:2.2.1";
```

Comment:
No - old is coming from `Cordova CLI` generated app, not from our libs. In this version, `Cordova` is still on `2.2.1`. In the next version, they've also moved to `3.3` (when we upgrade, we can get rid of this step).